client/mock/api/user/login.ts

```diff
@@ -0,0 +1,33 @@
+import { response } from '../_common/axios_response';
+import type { LoginRequestData, LoginResponse } from '../models/login_model';
+
+const mockLogin = (data: LoginRequestData): LoginResponse => {
+	const mockSessionIdBase = 'itsjustacookiefor_';
+
+	// You can login only with the same username - password.
+	const success = data.user === data.password;
+	const userId = getUserIdFromUsername(data.user);
```

Comment:
So when using the mock backend, username & password should look something like this - user_1 ?

Reply:
Yes, _anything__{userid}
It will save the userid to the sessionid